### PR TITLE
Improve Send Screen Design

### PR DESCRIPTION
**Pull request by NovacAndrei**
- Send Screen more responsive
- add iPhone X support

### CODE REVIEW

File: src/screens/send/styles.ts [hunk 1]
```diff
@@ -47,7 +54,11 @@ export default (theme: ITheme) =>
             backgroundColor: theme.colors.settingsDivider
         },
         bottomContainer: {
-            padding: BASE_DIMENSION + BASE_DIMENSION / 2,
+            padding: DEFAULT_BOTTOM_CONTAINER_PADDING,
+            paddingBottom: Platform.select({
+                default: DEFAULT_BOTTOM_CONTAINER_PADDING,
+                ios: isIphoneXorAbove() ? BASE_DIMENSION * 3 : DEFAULT_BOTTOM_CONTAINER_PADDING
```
Comment (krisboit) [inline src/screens/send/styles.ts:60]: doe this fix our layout issue on smaller devices ? Looking at the code to me it seems only ios is impacted by these changes...

also we should solve this by using flex functionality.

Comment (NovacAndrei) [reply src/screens/send/styles.ts:65]: I made some changes regarding the padding and margins. 
I have tested on both Pixel 2 and iPhone SE.

`Pixel 2`
image

`iPhone SE`
image

Also the scroll should work properly.
Please try again now with this code when you can.

Comment (krisboit) [reply src/screens/send/styles.ts:65]: I think the problem appears on testnet, when the yellow badge is visible

Comment (NovacAndrei) [reply src/screens/send/styles.ts:65]: @krisboit It seems to work ok on Pixel 2 with testnet badge:

https://drive.google.com/open?id=1rwlVpX3lFA-eMu9aMfXfr5gCPp1mzFDc

**Pull request merged**